workers/unity/Packages/com.improbable.gdk.deploymentlauncher/DeploymentLauncherWindow.cs

```diff
@@ -83,11 +87,63 @@ private void Update()
                 else
                 {
                     var error = result.UnwrapError();
-                    Debug.LogError($"Launch of {wrappedTask.Context.Name} failed. Code: {error.Code} Message:{error.Message}");
+                    Debug.LogError($"Launch of {wrappedTask.Context.Name} failed. Code: {error.Code} Message: {error.Message}");
+
+                    if (error.Code == Ipc.ErrorCode.Unauthenticated)
+                    {
+                        Debug.LogError("Please login with \"spatial auth login\" and try again.");
```

Comment:
this is pretty troll... shouldn't we just do it for them if we realize that happens?

Reply:
We could do - but I'm not sure how to encode it nicely into the UI flow, I can take a look at it

Reply:
Had a stab at it - made the flow a little trickier - but works.

I reckon it _could_ be refactored again, but unsure if its worth it